frontend/src/styles/MainPage.css

```diff
@@ -37,6 +37,10 @@
   margin: 30px;
 }
 
+.categoryIcon {
```

Comment:
Ezt nem látom, hogy a most beküldött kódban bárhol is használva lenne.

Reply:
Done.